extensions/azcli/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "azcli",
   "displayName": "%azcli.arc.displayName%",
   "description": "%azcli.arc.description%",
-  "version": "1.5.2",
+  "version": "1.5.3",
```

Comment:
You shouldn't need to bump this unless you're actually making changes

Reply:
I bumped this because our team wants to differentiate between versions for test, preview, and release, so changing between test and preview would warrant bumping it up.

Reply:
@candiceye I am not sure I am following.